pandas/tests/series/test_diff.py

```diff
@@ -0,0 +1,54 @@
+from numpy import nan
+from pandas import Series, date_range
+from pandas.tests.series.common import TestData
+from pandas.core.indexes.timedeltas import TimedeltaIndex
+from pandas.util.testing import assert_series_equal
+
+
+class TestDiff(TestData):
```

Comment:
you can drop the class TestDiff
use a fixture for ts here (it’s likely already a fixture)

can you split this up into separate tests / paramaterize

Reply:
I would love to, but I will learn that after the merge, I do not have the time now to fix it. Rewriting the whole file will take a while, let alone that the results of the test should always be the same, which cannot be guaranteed after a rewrite.